scripts/irods/test/test_all_rules.py

```diff
@@ -1192,6 +1192,7 @@ def do_test(_entity_type):
         do_test('data_object')
         do_test('collection')
 
+    # Note topology tests are not typically run with PREP for release
```

Comment:
I guess these tests were already being skipped in 4-2-stable and this was the cherry-pick diff?

Reply:
Yes, they were being skipped in 4-2-stable and not main, apparently. Probably just a forgotten cherry-pick from some time back.